container/worker/run_openqa_worker.sh

```diff
@@ -9,8 +9,11 @@ fi
 /root/qemu/kvm-mknod.sh
 
 if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+  groupmod -g "$group" --non-unique kvm
   usermod -a -G kvm _openqa-worker
+  qemu-system-x86_64 -enable-kvm -S &
```

Comment:
this is arch-specific and should be avoided

Reply:
I provided a new way to check the qemu without using kvm